packages/components/src/components/list-box/_list-box.scss

```diff
@@ -535,7 +535,7 @@ $list-box-menu-width: rem(300px);
 
   .#{$prefix}--list-box.#{$prefix}--list-box--inline
     .#{$prefix}--list-box__menu-item__option {
-    margin: 0 rem(32px) 0 $carbon--spacing-03;
+    margin: 0 $carbon--spacing-03;
```

Comment:
the items are still cut off early, should we reduce the right padding by 8px as well? I think this will account for the margin and resolve the issue

![image](https://user-images.githubusercontent.com/8265238/74928428-18d8fa80-539f-11ea-8733-5735b7799af8.png)

with reduced right padding:

![image](https://user-images.githubusercontent.com/8265238/74927088-9d764980-539c-11ea-87f7-2979d5b5c7ad.png)

Reply:
Seems like removing the padding affects the border-bottom

Reply:
I'm not able to reproduce that but it shouldn't. just to clarify I am reducing (not removing) the padding. but if we were to remove the padding the width (and border) would still be unaffected

Reply:
Oh could have sworn that image had a border going all the way to the right 🙃

Reply:
yeah if it was just the image then it was a bad screenshot. you can test it locally by changing the padding in devtools. the margin is what will determine where the border is

Reply:
Yeah, I had just skimmed the PR and saw that. Now that I've checked it out I agree, we can reduce this to `1.5rem` and have the entire `Option` text show, but still leave space for the checkmark 

`padding-right: $carbon--spacing-06;`

Reply:
I'll make the change! I was hoping to get design insight to see if when they wanted the text to truncate but I think I'm probably good to just make the change